### ISSUE TITLE
添加选项以在隐藏应用程序菜单时切换菜单栏的显示/隐藏[Linux]

### ISSUE THREAD

**Issue by gitSaptarshi** [agalwood/Motrix#956]
**功能说明**
Motrix是Linux上最方便的下载器应用程序。默认的标题栏样式是Windows之类的窗口，在GNOME和Pantheon（在Elementary OS中使用）之类的桌面环境中，它看起来完全不合时宜和丑陋。如果要使用系统的本机标题栏，它还将引入一个菜单栏，该菜单栏不适用于许多Linux桌面环境。最好使用键盘快捷键打开/关闭菜单栏，这将使应用程序具有更原始的外观。

**解决方案**
添加键盘快捷键以切换菜单栏的显示/隐藏。

**额外的背景**
我正在使用Elementary OS，并创建了一个带有和不带有菜单栏的外观模型。
![motrix](https://user-images.githubusercontent.com/37685073/118765944-64779e00-b899-11eb-8408-ad311f567be4.png)

P.S. 我不懂中文，并且已使用Google翻译将中文（简体）形式的文本翻译成英文。

**Comment by agalwood** [comment]
偏好设置——  [√] 隐藏菜单栏（仅支持 Windows 和 Linux）
去掉 √ 保存并应用

**Comment by agalwood** [comment]
You can use English : )

**Comment by agalwood** [comment]
There are various Linux desktop environments, and the effects of the native title bar are also various. In my opinion, the best effect is that there is no title bar and menu bar, so it is hidden by default. The hidden menu bar can still trigger application functions through shortcut keys.

**Comment by gitSaptarshi** [comment]
Yeah, but when the menu bar is hidden the title bar looks more alien.

**Comment by agalwood** [comment]
Closing this as a legacy Motrix v1 UI/runtime request. Motrix v2 has a rebuilt Electron/React interface and platform integration stack, so the original implementation and screenshots no longer apply. Please open a focused v2 issue if the current application still has a specific actionable problem.
